index.md

```diff
@@ -16,49 +16,33 @@ order: 1
 ## Introduction
 {:#introduction}
 
-Web accessibility evaluation tools are software programs or online services. They help determine if a Web site is accessible. This document highlights features of evaluation tools. It discusses different aspects of evaluation tools to consider when selecting these tools. 
-
-Automated web accessibility evaluation tools are unable to test all accessibility criteria. Many aspects of accessibility need human verification. 
-
-The [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/) provides several features of evaluation tools. Some of these features include: 
-
-- Guidelines
-- Specific uses of the tool
-- What authoring tools integrate with the evaluation tool
-- API
-- Browser plugin
-- etc.
-
-This document also discusses some specific usage of evaluation tools, such as:
-- Reports
-- Step-by-step evaluation
-- In-page feedback
-- Page transformation
-
-Finally, this document offers further aspects to consider, such as:
-- Organizational structure and development process
-- Complexity and size of Web site
-- Skills and knowledge of Web developers
-
-*Disclaimer*: WAI encourages the development and evolution of Web accessibility evaluation tools. WAI maintains an extensive [list of evaluation, repair and transformation tools](https://www.w3.org/WAI/ER/tools/). WAI does not endorse or promote any single tool or vendor.
```

Comment:
I feel that the disclaimer is no longer needed on *this* page but happy to discuss.